dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:stack_trace/stack_trace.dart';
+
+import 'protocol/sentry_stack_frame.dart';
+
+class SentryStackTraceFactory {
+  List<SentryStackFrame> getStackFrames(dynamic stackTrace) {
```

Comment:
I don't see the `origin` field playing a role here

Reply:
From previous release, in a browser context, absPath should be `'$origin${_absolutePathForCrashReport(frame)}'`

Reply:
yep but if you didn't take it as a `param` how would you concatenate?
it could be also a ctor param btw as it doesnt change

Reply:
✅  the origin is now a SentryStackFrame ctor param